Add new email notification article

OtterleyW opened the pull request.

Gnito commented on src/docs/tutorial-transaction-process/add-email-notification/index.md:
```diff
@@ -0,0 +1,250 @@
+- _new-booking-request-for-customer-subject.txt_.
+
+Remember to follow the naming convention where the file ends with
+\*-html.html or \*-subject.txt. Otherwise, the notification files will
```
Does this work with italics?

```suggestion
_\*-html.html_ or _\*-subject.txt_. Otherwise, the notification files will
```

OtterleyW commented:
@Gnito when looking at the artice again I agree that not using extraInfo would be better because it hides quite important part. I also noticed there was some unused css so I removed that and added a simpler css example there.
